employees/common/exports.py

```diff
@@ -0,0 +1,140 @@
+from openpyxl import Workbook
+from openpyxl.styles import Alignment
+from openpyxl.styles import Font
+from openpyxl.utils import get_column_letter
+
+from employees.common.constants import ExcelGeneratorSettingsConstants
+from managers.models import Project
+from users.models import CustomUser
+
+
+def set_format_styles_for_main_cells(cell):
+    cell.font = Font(name=ExcelGeneratorSettingsConstants.FONT.value, bold=True)
+    cell.alignment = Alignment(horizontal=ExcelGeneratorSettingsConstants.CENTER_ALINGMENT.value)
+
+
+def set_columns_width(worksheet, col_num, width_settings):
+    column_letter = get_column_letter(col_num)
+    column_dimensions = worksheet.column_dimensions[column_letter]
+    column_dimensions.width = width_settings[col_num - 1]
+
+
+def fill_headers_for_single_user(worksheet):
```

Comment:
There are some methods that are duplicated for user and project:
fill_headers_for_single_user / fill_headers_for_reports_in_project
fill_data_in_reports_for_single_user / fill_data_in_reports_for_project
summarizing_reports / summarizing_reports_for_project
generate_xlsx_for_single_user / generate_xlsx_for_project

Could you please review them for a common parts that can be extracted to separate functions, or maybe the whole part can be a single method with just few params?

Reply:
During implementation I was trying to do it but I didn't find better option than I have written. Because our file with single user reports is different than for whole project. Project reports doesn't have for example Project column, because all reports are from project which generated our data. I could make it in one big method but there will be many if syntax and I think it could be more hard to read but I know that now it is not easy to read, too.

Reply:
I have tried to extract to separate functions. I don't have any better idea

Reply:
Looks very good, please try to make `fill_data_in_reports_for_single_user` and `fill_data_in_reports_for_project` single function by passing `storage_data` as parameter.
Please also make sure you have cleaned unused values from constants enums.

Reply:
I have added some solution

Reply:
Looks very good!